adobe/adobe-license-decoder.rs
# Fixes for issues reported from the field

## Description
Before, when looking for a locally cached license against an operating config, we accepted any cached license for the config's app that had the right cert group.  But multiple configs can license the same app, so that would use the same cached license for configs from different packages.  Now we check the package ID in the license against the package ID of the config before we use the license for that config.

Additionally, each type of FRL license (offline/isolated, online, LAN, etc.) uses a different certificate ID for caching its results.  The current implementation did not know what cert ID to use for which type of license, so only online cached licenses were reported.

Finally, as reported in #14, the package UUID reported by `adobe-license-decoder -v` does not match the package npdId as displayed in the Admin Console, so the language and info reported by the adobe-license-decoder needs to be updated.

## Testing instructions
Multi-package scenario:

1. Install two different FRL online license packages for the same product.
2. Launch an app licensed by the package to activate it and fetch a license.
3. Run `adobe-license-decoder -vv`.  The cached license should only be reported against the most-recently installed package (which is the one that will be used for activation).
4. Remove the most recently installed package.
5. Run `adobe-license-decoder -vv`.  No cached license should be reported.
6. Launch the product again.  This will activate the product against the remaining package.
7. Run `adobe-license-decoder -vv`.  The cached license should now be reported against the remaining package.

Non-online scenario:

1. Install an FRL isolated license for a product.
2. Launch an app licensed by the package to activate it and fetch a license.
3. Run `adobe-license-decoder -vv`.  The cached license should be reported (it wasn't before).

## Related Issue
Fixes #11 
Fixes #14 

## Tasks


- [x] I have signed the [CLA](http://adobe.github.io/cla.html).
